modules/core/src/controllers/first-person-controller.js

```diff
@@ -1,16 +1,16 @@
 import Controller from './controller';
 import ViewState from './view-state';
 
-import {Vector3, clamp} from 'math.gl';
+import {Vector3, _SphericalCoordinates as SphericalCoordinates, clamp} from 'math.gl';
 
-const MOVEMENT_SPEED = 1; // 1 meter per keyboard click
-const ROTATION_STEP_DEGREES = 2;
-
-/* Helpers */
-
-function ensureFinite(value, fallbackValue) {
-  return Number.isFinite(value) ? value : fallbackValue;
-}
+const MOVEMENT_SPEED = 20;
+const DEFAULT_STATE = {
+  position: [0, 0, 0],
+  pitch: 0,
+  bearing: 0,
+  maxPitch: 90,
+  minPitch: -90
+};
 
```

Comment:
Does `FirstPersonController` have a documentation?

Reply:
No. `FirstPersonController` is not exposed; it is only enabled via `controller: true` on `FirstPersonView`.